src/app/modules/shared/components/sidebar/sidebar.component.scss

```diff
@@ -65,7 +79,7 @@ body {
 }
 
 .dark-mode-toggle {
```

Comment:
This style is not used, it would be better to remove it.

Reply:
Thanks for your review Josue, I will delete it in the next commit.